internal/schemas.ts

```diff
@@ -1144,6 +1144,45 @@ const PointsAnnotation: FoxgloveMessageSchema = {
   ],
 };
 
+const TextAnnotation: FoxgloveMessageSchema = {
+  type: "message",
+  name: "TextAnnotation",
+  description: "A text label on a 2D image",
+  fields: [
+    {
+      name: "timestamp",
+      type: { type: "primitive", name: "time" },
+      description: "Timestamp of annotation",
+    },
+    {
+      name: "position",
+      type: { type: "nested", schema: Point2 },
+      description: "Center of the circle in 2D image coordinates (pixels)",
```

Comment:
This needs to be updated to specify where on the text annotation this is positioning by ie what's the anchor used to position the text

Reply:
Also on second look, should this use normalized image coordinates ([0-1])so that it doesn't depend on the resolution of the image?

Reply:
@snosenzo if you want to make a breaking change to all of the image annotations I'm ok with that. But it would be confusing to have one annotation use normalized coordinates and the rest use pixel coordinates.